replica_aggregator/src/test/scala/uk/ac/wellcome/platform/storage/replica_aggregator/ReplicaAggregatorFeatureTest.scala

```diff
@@ -42,6 +55,30 @@ class ReplicaAggregatorFeatureTest
             )
           )
 
+          val expectedReplicaPath =
+            ReplicaPath(payload.bagRootLocation.path)
+
+          val stored =
+            versionedStore.get(id = Version(expectedReplicaPath, 0)).right.value
+
+          stored.identifiedT.length shouldBe 1
+
+          val replicaResult = stored.identifiedT.head
+
+          inside(replicaResult) {
+            case ReplicaResult(
+                ingestId,
+                PrimaryStorageLocation(
+                  storageProvider,
+                  location
+                ),
+                _
```

Comment:
Maybe assert that the replica result time is recent?

Reply:
I feel that's a bit too fuzzy and ambiguity here might lead to confusion later. 

In that vein I do think it is worth asserting something about that value and not simply `_` it out (i'm going to assert it is an Instant for now).